README.md

```diff
@@ -13,12 +13,32 @@ Create and manage Azure Virtual Machines directly from VS Code.
 
 ### Create New Virtual Machine
 
-![CreateVirtualMachine](resources/CreateVirtualMachine.gif)
+- Create an Azure VM with the default image of "Ubuntu 18.04-LTS" and size "Standard D2s V3"
+
+![CreateVirtualMachine](resources/CreateAzureVM.png)
+
+![CreateVirtualMachine2](resources/CreateAzureVM2.png)
+
+- Automatically generates and configures an SSH key used to remotely connect to new Azure VM
+
+![CreateVirtualMachine3](resources/CreateAzureVM3.png)
 
 ### Add SSH key to existing Virtual Machine
 
-![AddSSHKey](resources/AddSSHKey.gif)
+- Add SSH public keys to existing Azure VMs
+
+![AddSSHKey](resources/AddSSHKey.png)
+
+![AddSSHKey2](resources/AddSSHKey2.png)
+- Automatically configures `.ssh/config` file to be used to SSH into Azure VM
+
+![AddSSHKey3](resources/AddSSHKey3.png)

```

Comment:
So when I think about screenshots, I feel like the main goal is to tell people how to do things. The second and third screenshots don't add much value in that sense. Users already know how to select a file (AddSSHKey2) and read a notification (AddSSHKey3)

I would suggest modeling after this: https://github.com/Microsoft/vscode-azureappservice#deploy-your-first-nodejs-app-to-azure (And I assume Matt would also suggest that since he wrote it 😂)

Similar feedback for `CreateAzureVM` pics